spec.html

```diff
@@ -42725,7 +42774,7 @@ <h1>Date.prototype.setYear ( _year_ )</h1>
       <emu-annex id="sec-date.prototype.togmtstring">
         <h1>Date.prototype.toGMTString ( )</h1>
         <emu-note>
-          <p>The property *"toUTCString"* is preferred. The *"toGMTString"* property is provided principally for compatibility with old code. It is recommended that the *"toUTCString"* property be used in new ECMAScript code.</p>
+          <p>The `toUTCString` method is preferred. The `toGMTString` method is provided principally for compatibility with old code.</p>
```

Comment:
ooh, this sounds like a great thing to mark "icky" once we figure that styling out :-D

Reply:
Universal, you'll always be Greenwich to me.